libraries/apollo-runtime/src/androidInstrumentedTest/kotlin/instrumented/NetworkMonitorTest.kt

```diff
@@ -37,7 +38,9 @@ class NetworkMonitorTest {
   @Test
   fun test() = mockServerTest(
       clientBuilder = {
-        networkMonitor(NetworkMonitor(InstrumentationRegistry.getInstrumentation().context))
+        retryOnErrorInterceptor(
+            RetryOnErrorInterceptor(NetworkMonitor(InstrumentationRegistry.getInstrumentation().context))
+        )
```

Comment:
I removed `ApolloClient.Builder.networkMonitor(NetworkMonitor)` as part of this PR. It makes the call site a bit more verbose but allow for better locality as the `NetworkMonitor` isn't used outside the `RetryOnErrorInterceptor`